cloud_storage_handler/clients/minio_client.py

```diff
@@ -50,7 +75,7 @@ def __init__(self):
             endpoint=f"{config.hostname}:{config.port}",
             access_key=config.access_key,
             secret_key=config.secret_key,
-            secure=False,
+            secure=config.secure,
```

Comment:
If the user has set up TLS certs from their minio deployement, they will turn on the security from config.yaml, this should not be hardcoded false, otherwise this would be way too insecure.